scripts/testing/install_dependencies.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Anaconda has plenty of dependencies and because of that it's hard to set
+# environment to with Anaconda properly.
+# Run this script to install all the required dependencies for the autotools to be
+# able to call make commands. This will enable you to test anaconda more easily.
+#
+# To avoid messing your system feel free to use `toolbox` in fedora so all the
+# dependencies will be installed to the container instead of your system.
+# The toolbox package is in standard Fedora repository.
+#
+# For toolbox use case run:
+#
+#   $ toolbox create
+#   $ toolbox enter
+#   $ dnf install rpm-build
+#   $ sudo ./scripts/testing/install_dependencies.sh
+#
+#
+# For direct host installation run:
+#
+#   $ sudo dnf install rpm-build  # to get rpmspec binary
+#   $ sudo ./scripts/testing/install_dependencies.sh
+#
+
+
+# remove all problematic pieces from anaconda spec to be able to get dependencies
+cat ./anaconda.spec.in | sed 's/@PACKAGE_VERSION@/0/; s/@PACKAGE_RELEASE@/0/; s/%{__python3}/python3/' > /tmp/anaconda.spec
+
+# get all build requires dependencies from the spec file and strip out version
+# version could be problematic because of fedora version you are running and
+# they are mostly not important for automake
+rpmspec -q --buildrequires /tmp/anaconda.spec | sed 's/>=.*$//' | xargs -d '\n' dnf install
```

Comment:
I suggest to append `$@` here. That way, one can call the script as `install_dependencies.sh -y` and make it non-interactive. An usual workflow is to rebuild your toolboxes regularly and often, and interactive questions are annoying.

Reply:
Good idea.